msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,10 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot the first time it's added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. After a team member is added, the bots receives unique channel ID as a conversation ID using the [conversationUpdate](#conversation-update-events) and [installationUpdate](#installation-update-event) events. You can also cache for future use, such as sending a message to a specific user.bots. The bot posts welcome message in the same channel by default instead.
+
+> [!NOTE]
+> The conversation ID is set to the user selected channel.
```

Comment:
It would help to have a reminder here on how to get the team information, now that the conversation ID points to the selected channel. (get from the team info in channel data)

Reply:
Updated.